### PR TITLE
Find references to fields via getters/setters

### PR DESCRIPTION
**Pull request by snjeza**
See eclipse/eclipse.jdt.ls#1548
Requires https://github.com/eclipse/eclipse.jdt.ls/pull/1561

Signed-off-by: Snjezana Peco <snjezana.peco@redhat.com>

### CODE REVIEW

**Review by rgrunber — COMMENTED** [review]
@fbricon should we wait a day for results of https://twitter.com/VSCodeJava/status/1314145144791236608 , or are you fine with disabled by default ?

Change looks fine to me overall.

**Comment by rgrunber** [comment]
Whoops, meant to re-review myself. @snjeza I think this is fine to merge. We can always revisit the default value at a later time.

File: README.md [hunk 1]
```diff
@@ -171,6 +171,9 @@ The following settings are supported:
 * `java.templates.fileHeader`: Specifies the file header comment for new Java file. Supports configuring multi-line comments with an array of strings, and using ${variable} to reference the [predefined variables](https://github.com/redhat-developer/vscode-java/wiki/Predefined-Variables-for-Java-Template-Snippets).
 * `java.templates.typeComment`: Specifies the type comment for new Java type. Supports configuring multi-line comments with an array of strings, and using ${variable} to reference the [predefined variables](https://github.com/redhat-developer/vscode-java/wiki/Predefined-Variables-for-Java-Template-Snippets).
 
+New in 0.68.0:
+* `java.references.includeGetterSetter`: Include getter, setter and builder/constructor when finding references.
```
[the remark this answers is not in the document]
Comment (fbricon) [reply README.md:175]: default to true.

File: package.json [hunk 2]
```diff
@@ -641,6 +641,12 @@
           "markdownDescription": "Specifies the type comment for new Java type. Supports configuring multi-line comments with an array of strings, and using ${variable} to reference the [predefined variables](command:_java.templateVariables).",
           "scope": "window",
           "default": []
+        },
+        "java.references.includeGetterSetter": {
+          "type": "boolean",
+          "default": false,
```
Comment (fbricon) [inline package.json:647]: true

The people has spoken https://twitter.com/VSCodeJava/status/1314145144791236608

File: README.md [hunk 3]
```diff
@@ -171,6 +171,9 @@ The following settings are supported:
 * `java.templates.fileHeader`: Specifies the file header comment for new Java file. Supports configuring multi-line comments with an array of strings, and using ${variable} to reference the [predefined variables](https://github.com/redhat-developer/vscode-java/wiki/Predefined-Variables-for-Java-Template-Snippets).
 * `java.templates.typeComment`: Specifies the type comment for new Java type. Supports configuring multi-line comments with an array of strings, and using ${variable} to reference the [predefined variables](https://github.com/redhat-developer/vscode-java/wiki/Predefined-Variables-for-Java-Template-Snippets).
 
+New in 0.68.0:
```
[the remark this answers is not in the document]
Comment (snjeza) [reply README.md]: Fixed.

File: package.json [hunk 4]
```diff
@@ -641,6 +641,12 @@
           "markdownDescription": "Specifies the type comment for new Java type. Supports configuring multi-line comments with an array of strings, and using ${variable} to reference the [predefined variables](command:_java.templateVariables).",
           "scope": "window",
           "default": []
+        },
+        "java.references.includeGetterSetter": {
```
[the remark this answers is not in the document]
Comment (snjeza) [reply package.json]: Fixed.

File: package.json (hunk 2 above)
Comment (snjeza) [reply package.json]: Fixed.

**Pull request merged**